tests/dummy/app/templates/button.hbs

```diff
@@ -8,12 +8,13 @@
 {{/paper-toolbar}}
 {{#paper-content class="md-padding demo-buttons"}}
 <div class="doc-content">
-  <p>
+  <div layout="row" class="flex">
```

Comment:
Well, this uses the layout attribute and a flex class. There is a `layout-row` *I think*. I've used it in the toolbar page.